Jenkinsfile

```diff
@@ -364,15 +364,20 @@ pipeline {
       }
     }
     stage('Deploy') {
+      when { branch 'master' }
       agent {
         dockerfile {
           additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
+          args '-v /var/run/docker.sock:/var/run/docker.sock'
```

Comment:
what is this doing?

Reply:
This allows this docker container to speak to the LXC container docker daemon directly. Launching a docker daemon inside a docker container is strongly discouraged (see https://jpetazzo.github.io/2015/09/03/do-not-use-docker-in-docker-for-ci/), instead you are supposed to launch it using the hosts docker daemon, which this enables.

Note this is better for cacheing the images that are build inside this container too.